src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -51,18 +51,18 @@ public static Name parseName(String name) throws ParseException {
     }
 
     /**
-     * Parses a {@code String phone} into a {@code Phone}.
+     * Parses a {@code String phone} into a {@code Amount}.
      * Leading and trailing whitespaces will be trimmed.
      *
      * @throws ParseException if the given {@code phone} is invalid.
      */
-    public static Phone parsePhone(String phone) throws ParseException {
+    public static Amount parsePhone(String phone) throws ParseException {
```

Comment:
`parsePhone` should be renamed to `parseAmount` with the corresponding parameter renamed. Same for all other instances.